tests/test_examples.py

```diff
@@ -0,0 +1,46 @@
+import glob
+import os
+import shutil
+
+import pytest
+
+from rapids_dependency_file_generator.rapids_dependency_file_generator import (
+    main as dfg,
+)
+
+CURRENT_DIR = os.path.dirname(__file__)
+
+
+@pytest.fixture(scope="session", autouse=True)
+def clean_actual_files():
+    for root, _, _ in os.walk("tests"):
+        if os.path.basename(root) == "actual":
+            shutil.rmtree(root)
+
+
+def test1():
+    test_dir = os.path.join(CURRENT_DIR, "examples", "test1")
```

Comment:
Use a parametrize here so it's easy to add future tests.
```suggestion
@pytest.mark.parametrize(
    "test_dir",
    ["test1"],
)
def test1(test_dir):
    test_dir = os.path.join(CURRENT_DIR, "examples", test_dir)
```